documents/addon.md

```diff
@@ -0,0 +1,179 @@
+## Addons
+
+### Create an addon
+
+```go
+subscriptionId := "sub_00000000000001"
+data := map[string]interface{}{
+  "item": map[string]interface{}{
+      "name":"Extra appala (papadum)",
+      "amount":3000,
+      "currency":"INR",
```

Comment:
description can also be present in items. 
https://razorpay.com/docs/api/payments/subscriptions/#request-parameters-6